src/dashboard/Settings/Settings.js

```diff
@@ -56,29 +57,84 @@ export default function Settings(props) {
               '0xea718e4602125407fafcb721b7d760ad9652dfe7',
               'kovan'
             )}{' '}
+            <CopyToClipboard
+              text={'0xea718e4602125407fafcb721b7d760ad9652dfe7'}
+            >
+              <Button
+                style={{
+                  margin: '10px 0 10px',
+                  backgroundColor: '#1b1c4c',
+                  marginLeft: 10
+                }}
+                variant="contained"
+                color="primary"
+              >
+                Copy
+              </Button>
+            </CopyToClipboard>
             <br />
             DAI:{' '}
             {getEtherscanLink(
               '0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99',
               'kovan'
-            )}{' '}
-            0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99
+            )}
+            <CopyToClipboard
+              text={'0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99'}
+            >
+              <Button
+                style={{
+                  margin: '10px 0 10px',
+                  backgroundColor: '#1b1c4c',
+                  marginLeft: 10
+                }}
+                variant="contained"
+                color="primary"
+              >
+                Copy
+              </Button>
+            </CopyToClipboard>
             <br />
             cDAI:{' '}
             {getEtherscanLink(
               '0x0a1e4d0b5c71b955c0a5993023fc48ba6e380496',
               'kovan'
             )}
+            <CopyToClipboard
+              text={'0x0a1e4d0b5c71b955c0a5993023fc48ba6e380496'}
+            >
+              <Button
+                style={{
```

Comment:
there are 3 buttons here that have the same code. We should start thinking about how to reuse these components